java-11/Dockerfile

```diff
@@ -41,8 +43,8 @@ java.net.http,\
 jdk.zipfs\
  --output jre
 
-# We extract JRE's hard dependencies, libz and SSL certs, from the fat JRE image.
-FROM gcr.io/distroless/java:11-debug AS deps
+# Extract java SSL certs
+FROM gcr.io/distroless/java:11-debug AS ssl
```

Comment:
Since this isn't java version dependent (as made clear by the fact we're using it for 21 too), can we use `gcr.io/distroless/java-base-debian12:debug` instead of a specific jdk version?